### PR TITLE
Update dependencies and switch to new es2015 syntax

### PR DESCRIPTION
**Pull request by ionphractal**
Latest Electron gives me a little of a performance boost. I also updated the rest of the dependencies and fixed upcoming linter errors.

### CODE REVIEW

File: app/containers/zazu.js [hunk 1]
```diff
@@ -1,66 +1,59 @@
 const React = require('react')
+const PropTypes = require('prop-types')
```
Comment (bayleedev) [inline app/containers/zazu.js:2]: I don't see this in package.json. Also could this instead be:
```js
const { PropTypes } = require('react')
```

File: app/containers/zazu.js [hunk 2]
```diff
@@ -80,7 +73,17 @@ const Zazu = React.createClass({
           />
       </div>
     )
-  },
-})
+  }
+}
+
+Zazu.propTypes = {
```
Comment (bayleedev) [inline app/containers/zazu.js:79]: Can these be defined statically on the class?

File: app/lib/github.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ const currentRemoteVersion = (name) => {
     host: 'api.github.com',
     path: '/repos/' + name + '/commits',
   }).then((response) => {
-    return response[0] && response[0].sha || 'master'
+    return response[0] ? response[0].sha : 'master'
```
Comment (bayleedev) [inline app/lib/github.js:17]: I like this a lot better! :+1:

**Comment by bayleedev** [comment]
Great work!

File: app/containers/zazu.js (hunk 1 above)
Comment (ionphractal) [reply app/containers/zazu.js:2]: Ah, just forgot to add the module. Strange it didn't give me an error about it... Using PropTypes from React is afaik deprecated.

File: app/containers/zazu.js (hunk 2 above)
Comment (ionphractal) [reply app/containers/zazu.js:79]: Yes it works like that https://facebook.github.io/react/docs/typechecking-with-proptypes.html

**Comment by ionphractal** [comment]
Hmm.. this travis error looks like a temporary timeout. I don't have it on my machine except when I do other things while running cucumber.

**Comment by bayleedev** [comment]
Thank you!

**Pull request merged**